test/e2e/parallel/spacebindingrequest_test.go

```diff
@@ -123,36 +122,26 @@ func TestCreateSpaceBindingRequest(t *testing.T) {
 
 			// then
 			// wait for spacebinding request status to be set
-			_, err = memberAwait.WaitForSpaceBindingRequest(t, types.NamespacedName{Namespace: spaceBindingRequest.GetNamespace(), Name: spaceBindingRequest.GetName()},
+			_, err = memberAwait.WaitForSpaceBindingRequest(t, client.ObjectKeyFromObject(spaceBindingRequest),
 				UntilSpaceBindingRequestHasConditions(spacebindingrequesttestcommon.UnableToCreateSpaceBinding(fmt.Sprintf("invalid role 'invalid' for space '%s'", space.Name))),
 			)
 			require.NoError(t, err)
 			bindings, err := hostAwait.ListSpaceBindings(space.Name)
 			require.NoError(t, err)
 			assert.Len(t, bindings, 1)
-		})
 
-		t.Run("unable create space binding request with invalid MasterUserRecord", func(t *testing.T) {
```

Comment:
I'm not 100% sure I understand why dropping this test case. This is not updating the MUR but creating an SBR with an invalid MUR, which is not covered by the WebHook.

Reply:
already discussed in https://redhat-internal.slack.com/archives/CHK0J6HT6/p1716974045574919?thread_ts=1716912003.579899&cid=CHK0J6HT6
We cannot reset the MUR back to a valid value (webhook blocks that) so we would end up with an invalid SBR so we could face the same flakiness again. Let's rely on unit-tests in this particular situation